src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -13,11 +13,12 @@ def get(self, request, organization):
         providers = []
         for provider_id in provider_bindings:
             provider = provider_bindings.get(provider_id)(id=provider_id)
-            providers.append({
-                'id': provider_id,
-                'name': provider.name,
-                'config': provider.get_config(),
-            })
+            if provider.name != 'Bitbucket' or bool(organization.flags.early_adopter):
```

Comment:
it just makes it clearer what the feature is and follows a pattern we always use. it will make this line be something like `if features.has('bitbucket-repos')` which is a lot nicer and easier to search for / remove when we need to.